libs/sdk-ui-kit/src/Header/generateHeaderHelpMenuItems.ts

```diff
@@ -0,0 +1,95 @@
+// (C) 2020-2021 GoodData Corporation
+import { IHeaderMenuItem } from "./Header";
+import { ISettings } from "@gooddata/sdk-backend-spi";
+import { generateSupportUrl } from "../utils/featureFlags";
+
+type TUTMContent =
+    | "main_menu_help_documentation"
+    | "main_menu_help_university"
+    | "main_menu_help_community"
+    | "main_menu_help_support"
+    | "main_menu_help_ticket";
+
+/**
+ * @internal
+ */
+const addUTMParameters = (baseUrl: string, utmContent: TUTMContent, isBranded = false): string =>
+    isBranded
+        ? baseUrl
+        : `${baseUrl}${
+              baseUrl.includes("?") ? "&" : "?"
+          }utm_medium=platform&utm_source=product&utm_content=${utmContent}`;
+
+/**
+ * @internal
+ */
+export function generateHeaderHelpMenuItems(
+    documentationUrl?: string,
+    supportForumUrl?: string,
+    userEmail?: string,
+    workspaceId?: string,
+    sessionId?: string,
+    supportEmail?: string,
+    isBranded?: boolean,
+    featureFlags: ISettings = {},
+): IHeaderMenuItem[] {
+    const universityUrl = "https://university.gooddata.com";
+    const communityUrl = "https://community.gooddata.com";
+    const { enableUniversityHelpMenuItem, enableCommunityHelpMenuItem } = featureFlags;
+    const helpMenuItems: IHeaderMenuItem[] = [];
+
+    if (documentationUrl) {
+        helpMenuItems.push({
+            key: "gs.header.documentation",
+            className: "s-documentation",
+            href: addUTMParameters(documentationUrl, "main_menu_help_documentation", isBranded),
+            target: "_blank",
+        });
+    }
+
+    if (!!enableUniversityHelpMenuItem && !isBranded) {
```

Comment:
Nitpick: `!!` is not needed in `if`s. There are multiple occurences of this in this function.

Reply:
I removed the `!!` in this file thanks